<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>989</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>현재 날씨 - 대체로 맑음</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="ParentGridLayout">
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="letfTopGrid">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="mainPictureLabel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>290</height>
           </size>
          </property>
          <property name="text">
           <string>사진이 들어갈 자리</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="mainWeatherDesLabel">
          <property name="font">
           <font>
            <family>나눔바른고딕 UltraLight</family>
           </font>
          </property>
          <property name="text">
           <string>대체로 맑음</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="rightTop">
      <item>
       <widget class="QLabel" name="mainTempLabel">
        <property name="font">
         <font>
          <family>나눔바른고딕 Light</family>
          <pointsize>35</pointsize>
         </font>
        </property>
        <property name="text">
         <string>10°C</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="mainAdditionalInfoLabel">
          <property name="font">
           <font>
            <family>나눔바른고딕 UltraLight</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>습도 80%                                                               바람 0.3 km/h @ 20</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="mainHiLowTempLabel">
          <property name="font">
           <font>
            <family>나눔바른고딕 UltraLight</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>7°C / 19°C</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QScrollArea" name="scrollArea">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>300</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>965</width>
         <height>298</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
